lib/paypal_express/api.rb

```diff
@@ -187,16 +199,43 @@ def reset_payment_methods(kb_account_id, payment_methods, properties, context)
 
       def build_form_descriptor(kb_account_id, descriptor_fields, properties, context)
         # Pass extra parameters for the gateway here
-        options           = {}
-        properties        = merge_properties(properties, options)
-
-        # Add your custom static hidden tags here
-        options           = {
-            #:token => config[:paypal-express][:token]
-        }
-        descriptor_fields = merge_properties(descriptor_fields, options)
+        options              = {}
+        properties           = merge_properties(properties, options)
+        descriptor_fields    = merge_properties(descriptor_fields, options)
+        form_fields          = properties_to_hash(descriptor_fields)
+        descriptor           = super(kb_account_id, descriptor_fields, properties, context)
+        kb_account           = @kb_apis.account_user_api.get_account_by_id(kb_account_id, context)
+        kb_payment_method_id = (@kb_apis.payment_api.get_account_payment_methods(kb_account_id, false, [], context).find { |pm| pm.plugin_name == 'killbill-paypal-express' }).id
+        amount               = (form_fields[:amount] || "0").to_i
```

Comment:
I was looking at other plugins (such as https://github.com/killbill/killbill-payu-latam-plugin/blob/master/lib/payu_latam/api.rb#L215) and they seem to use `:amount` ?